src/lib/customers/xovis-de.json

```diff
@@ -0,0 +1,59 @@
+			},
+			{
+				"title": "Infrastruktur-Automatisierung",
+				"body": "Eine vollständige Automatisierung vom ersten Tag an war für Xovis von grösster Bedeutung, da das SaaS-Produkt an vielen Kundenflughäfen auf der ganzen Welt zur Verfügung gestellt werden sollte. Dies bedeutete, dass der identische Azure-Infrastruktur-Stack bei Bedarf in verschiedenen Azure-Regionen bereitgestellt werden musste, um die Latenzzeiten von Kundenstandorten zu reduzieren. bespinian unterstützte Xovis bei der Konzeption und Implementierung geeigneter Terraform-Module und Azure DevOps-Pipelines, um diesen Bereitstellungsprozess zu automatisieren."
+			},
+			{
+				"title": "Kubernetes-Automatisierung",
+				"body": "Gleichzeitig musste in jeder Region eine beliebige Anzahl von Kundenstandorten automatisch als Sammlung von Kubernetes-Deployments bereitgestellt werden. bespinian arbeitete mit dem Xovis-Team zusammen, um einen GitOps-Prozess auf der Grundlage von Helm und Argo CD zu entwerfen und zu implementieren. Dieser Prozess musste die Kundenstandortbereitstellung vollständig automatisieren, bis zum Punkt, an dem ein neuer Kunde vom Service Delivery-Team von Xovis in einem Hands-off-Prozess aufgesetzt werden konnte."
+			},
+			{
+				"title": "Containerisierung von Applikationen",
+				"body": "Während Xovis bereits Container für Entwicklungszwecke verwendete, bedeutete der Übergang zu einem Cloud-nativen Ansatz, dass Container zu den primären Artefakten des gesamten Entwicklungsprozesses werden mussten. bespinian arbeitete mit dem Xovis-Team zusammen, um bestehende Build-Pipelines in mehrstufige Image-Builds umzuwandeln, was zu maximal effizienten Runtime-Images für alle am SaaS-Produkt beteiligten Komponenten führte."
+			},
+			{
+				"title": "Zero-Downtime-Architektur",
+				"body": "Da viele Komponenten der Flughafensoftware von Xovis zustandsvolle Algorithmen, basierend auf Sensoreingaben über die Zeit ausführen, war es eine zusätzliche Herausforderung, diese Komponenten in die Lage zu versetzen, Upgrades und Wartungsarbeiten ohne Ausfallzeiten zu unterstützen. bespinian beriet das Xovis-Team bei der Entwicklung eines Stateful Load Balancing- und Failover-Schemas auf der Grundlage des HashiCorp Consul KV-Speichers und von Sessions, um sicherzustellen, dass einzelne Replikate von Komponenten einen Konsens darüber erzielen können, welche Sensoren verarbeitet werden sollen, und dass sie im Falle von Ausfällen während der Sensorverarbeitung sofort ersetzt werden können."
+			},
+			{
+				"title": "Security-Härtung",
```

Comment:
This sounds like an odd translation to me. I would just leave "Security Hardening" or put something like "Security Verbesserungen".

Reply:
Agreed. Done.